### ISSUE TITLE
Two datepickers can be opened at the same time

### ISSUE THREAD

**Issue by thaihacong** [kekeh/ngx-mydatepicker#5]
Hi @kekeh, could you please take the fix from mydatepicker 1.6.12 there.
https://github.com/kekeh/mydatepicker/issues/185

In ngx-mydatepicker 0.0.8, I can open two datepickers and both of them are displayed.

**Comment by kekeh** [comment]
I fixed this bug to version 0.0.9. Update to version 0.0.9.

Now when you call functions **openCalandar**, **toggleCalendar** or **clearDate** you can remove the **$event.stopPropagation()** call and do like below:
```html
<form>
    <div class="input-group">
        <input class="form-control" style="float:none" placeholder="Select a date" 
            ngx-mydatepicker name="mydate" [(ngModel)]="model" [options]="myOptions" 
            #dp="ngx-mydatepicker" (dateChanged)="onDateChanged($event)"/>

        <span class="input-group-btn">
            <button type="button" class="btn btn-default" (click)="dp.clearDate()">
                <i class="glyphicon glyphicon-remove"></i>
            </button>
            <button type="button" class="btn btn-default" (click)="dp.toggleCalendar()">
                <i class="glyphicon glyphicon-calendar"></i>
            </button>
        </span>
    </div>
</form>
```

**Comment by thaihacong** [comment]
Thank you very much for your quick fix. It works well now.
